### ISSUE TITLE
Polymer 2 migration?

### ISSUE THREAD

**Issue by kaceo** [Link2Twenty/pull-to-action#21]
I am developing under Polymer2 and caught this runtime error:

paper-spinner-behavior.html:65 Uncaught TypeError: this.getPropertyInfo is not a function
    at HTMLElement.__altChanged (paper-spinner-behavior.html:65)
    at Object.runObserverEffect [as fn] (property-effects.html:214)
    at runEffectsForProperty (property-effects.html:160)
    at runEffects (property-effects.html:126)
    at HTMLElement._propertiesChanged (property-effects.html:1647)
    at HTMLElement._flushProperties (property-accessors.html:549)
    at HTMLElement.ready (property-effects.html:1605)
    at HTMLElement.ready (element-mixin.html:633)
    at HTMLElement.ready (legacy-element-mixin.html:176)
    at HTMLElement.ready (class.html:215)

I think there is some breaking change to Polymer 2 that needs updating.

**Comment by Link2Twenty** [comment]
Yeah, I'd like to do a total rewrite at some point but don't really have the time at the moment,
